libs/cart/src/effects/cart.effects.spec.ts

```diff
@@ -128,35 +187,33 @@ describe('Daffodil | Cart | CartEffects', () => {
       });
     });
   });
-  describe('when CartResetAction is triggered', () => {
-
+  describe('when CartClearAction is triggered', () => {
     let expected;
-    const resetAction = new DaffCartReset();
+    const resetAction = new DaffCartClear();
```

Comment:
`resetAction` should be clearAction

Reply:
Changed in [`2eea542`](https://github.com/graycoreio/daffodil/pull/681/commits/2eea54244a04f5331df801d853e83d2dd2c74ab2).